src/cascadia/TerminalSettingsModel/CascadiaSettings.h

```diff
@@ -98,6 +98,12 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
         winrt::guid GetProfileForArgs(const Model::NewTerminalArgs& newTerminalArgs) const;
 
+        void RefreshDefaultTerminals();
+
+        Windows::Foundation::Collections::IObservableVector<Model::DefaultTerminal> DefaultTerminals() const noexcept;
+        Model::DefaultTerminal CurrentDefaultTerminal() const noexcept;
+        void CurrentDefaultTerminal(Model::DefaultTerminal terminal);

```

Comment:
```suggestion
        WINRT_PROPERTY(Model::DefaultTerminal, CurrentDefaultTerminal);
```
This macro creates a standard getter/setter/member for you.

Reply:
Can't do this either. There's no parameterless constructor and I don't technically know, inside Terminal, that the null one is conhost. That's handled by propslib.